### PR TITLE
CLUS-ucsd-rest-api103-md2

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* added periods
* changed dashes to em dashes
* changed single bullet lists to
   sentences
* removed capitals

### CODE REVIEW

File: labs/ucsd-rest-api-103/2.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# Introductory UCS Director REST API, Custom Tasks and Workflow Creation Part III
+# Introductory UCS Director REST API, custom tasks and workflow creation part III
```
Comment (paulzim) [inline labs/ucsd-rest-api-103/2.md]: Serial comma... "...REST API, custom tasks, and workflow creation part III"

File: labs/ucsd-rest-api-103/2.md [hunk 2]
```diff
@@ -83,70 +83,66 @@
...
   ![](/posts/files/ucsd-rest-api-103/assets/images/ucsd-rest-api-103-23.jpg)<br/><br/>
 
   ![](assets/images/ucsd-rest-api-103-23.jpg)<br/><br/>
 
-  8. Validate and Save the Workflow
+  8. Validate and Save the Workflow:
 
-    - ***Click*** - "Validate" icon, "The workflow is valid" message will appear
-    - ***Click*** - "X" in the *upper right-hand* corner to close the "Workflow Designer"<br/><br/>
+    - ***Click*** — "Validate" icon. "The workflow is valid" message appears.
```
Comment (paulzim) [inline labs/ucsd-rest-api-103/2.md]: Great! Nice catch on the tense.

File: labs/ucsd-rest-api-103/2.md (hunk 1 above)
Comment (ReneeBornstein) [reply labs/ucsd-rest-api-103/2.md]: I did this change, and others that Anne has had me do on all docs:
* made ***word*** to word
* made "word" to **word**
* removed doubled images

Comment (ReneeBornstein) [reply labs/ucsd-rest-api-103/2.md]: These changes will need to be done on md3 as well. I might have already done it, but let me know if I need to.

**Comment by annegentle** [comment]
@charleslavoy testing... @ReneeBornstein it’s likely we are simply seeing how GitHub works for notifications, I believe. You cannot “@“ someone who is not already associated with the repo in some way.

**Comment by annegentle** [comment]
@charleslavoy okay @ReneeBornstein I needed to add the repo to the team repo list. That is a good indicator that a repo is not properly included in the team repo list, the lack of auto fill on notify commands.

**Comment by charleslavoy** [comment]
@annegentle FYI ... I received your notification.

**Pull request merged**